### PR TITLE
Add post for quantumalgorithms.org

### PR DESCRIPTION
**Pull request by Scinawa**
I made two different commits, in the case you only want to host here html files, and you don't want to keep here the markdown.

### CODE REVIEW

**Review by nathanshammah — CHANGES_REQUESTED** [review]
Thanks @Scinawa, looks great to me! Minor points"
- A link is needed in the `blog.html` index to link the file there
- The date should be updated to Nov 1st (or when it gets updated, if later)
- supported by unitary.fund --> Unitary Fund
- Some suggestions to make it a bit less emphatic: "Suggestion: "Now you can learn everything about quantum algorithms on https://quantumalgorithms.org!" --> "Now you can learn more about quantum algorithms on https://quantumalgorithms.org."" This is just a suggestion. 

Green light from me once the small edits are addressed. Checking with @crazy4pi314 overall for homogeneity with other blog posts.

File: posts/_markdown/quantumalgorithmsorg.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+
+The book has a core team and many contributors that help keep it updated, clear, and correct. 
+The book started in its early days from the old blog of Alessandro Luongo, subsequently including parts of his Ph.D. thesis on quantum machine learning. Then we added contents from the MSc thesis of Armando Bellante. We recently hosted five students from the mentorship program of the Quantum Open Source Foundation. With them, we worked on quantum algorithms on graphs, quantum algorithms for speeding up Monte Carlo techniques, and quantum perceptrons algorithms. Many friends and great researchers have contributed to the project by spotting typos, writing chapters, submitting parts of their papers or thesis chapters. We wholeheartedly thank them and report their names here: Patrick Rebentrost, Yassine Hamoudi, Martin Plávala, Trong Duong, Filippo Miatto, Jinge Bao, Michele Vischi, Adrian Lee, Samantha Buck, Sathyawageeswar Subramanian.
```
Comment (crazy4pi314) [inline posts/_markdown/quantumalgorithmsorg.md:24]: Maybe vary the starts of the sentences on l23 and l24

File: posts/_markdown/quantumalgorithmsorg.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Quantumalgorithms.org has constantly been growing. The project is currently supported by the unitary.fund and the Centre For Quantum Technologies (the CQT of Singapore). These funds are dedicated to compensating collaborators that help us write more high-quality content and enrich the book's quality. At the moment, two collaborators are working on some brand-new, thrilling chapters that should be available by the end of the year!
```
Comment (crazy4pi314) [inline posts/_markdown/quantumalgorithmsorg.md:26]: I'd also hyper link QCT

```suggestion
[Quantumalgorithms.org](https://quantumalgorithms.org) has constantly been growing. The project is currently supported by the [Unitary Fund](https://unitary.fund) and the Centre For Quantum Technologies (the CQT of Singapore). These funds are dedicated to compensating collaborators that help us write more high-quality content and enrich the book's quality. At the moment, two collaborators are working on some brand-new, thrilling chapters that should be available by the end of the year!
```

File: posts/_markdown/quantumalgorithmsorg.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+ 
+We are eagerly waiting for different contributors: Typo Hunters, Friendly Reviewers, Consistency Fanatics, and Happy Writers. To name some tasks, we are searching for contributors to improve the section on error propagation and write more introductory material in chapters 2 and 3. Another much-appreciated contribution would be to give us feedback on the book's content: are there parts that are too hard to understand? Can a section be made more easily accessible? Are we missing some algorithm or paper to cite? We also have a list of exercises that are waiting to be added to the book. Are you willing to go the extra mile, write a whole new chapter, and get paid for your contribution? Contact us, and let's have a chat! We are waiting for you!
```
Comment (crazy4pi314) [inline posts/_markdown/quantumalgorithmsorg.md:32]: I think this comes off not how you intend, maybe something like "As the book grows, we are encouraging all kinds of contributors:...

Also, I am not quite sure what a happy writer is, :)

Making the tasks a list may make it easier to brows/catch people's attention.

**Review by crazy4pi314 — CHANGES_REQUESTED** [review]
LGTM, had a couple of comments on wording/links. Great post folks!

**Comment by crazy4pi314** [comment]
@Scinawa I forgot to ping you earlier, but if you want to have a look at the changes we can then get this posted!

File: posts/_markdown/quantumalgorithmsorg.md (hunk 1 above)
Comment (Scinawa) [reply posts/_markdown/quantumalgorithmsorg.md]: Changed in the new commit! :)

File: posts/_markdown/quantumalgorithmsorg.md (hunk 3 above)
Comment (Scinawa) [reply posts/_markdown/quantumalgorithmsorg.md]: Done! :) Thank you again!

**Comment by crazy4pi314** [comment]
@Scinawa the blog.html is not auto-generated, your addition is perfect! Merging, will put out socials tomorrow!

**Review by crazy4pi314 — APPROVED** [review]
LGTM!

**Comment by nathanshammah** [comment]
@Scinawa glad to see this merged!

**Comment by nathanshammah** [comment]
@Scinawa any help in improving the information on how to contribute to the blog is welcome, it's the first time we test it. If there is something unclear, please consider opening an issue or a PR to edit the Readme.

**Pull request merged**